netdata-installer.sh

```diff
@@ -13,8 +13,14 @@ fi
 LC_ALL=C
 umask 022
 
+DEFCFLAGS=
+case "$((arch))" in
```

Comment:
should be `$(arch)` otherwise it would be math expression from `$arch` variable which doesn't exist and the result is `0`